packages/itwinui-css/vite.config.ts

```diff
@@ -17,6 +17,9 @@ export default defineConfig({
       },
     },
   },
+  css: {
+    postcss: { plugins: [require('autoprefixer')] }
```

Comment:
Does it take our `.browserlistrc` since it is in the root?

Reply:
I'm not sure but I don't think it matters much, because this is just for the dev server.

Reply:
It matters because I think nobody checks the result with "prod" build.

Reply:
Visual tests still run on the prod build, and the github pages deployment also uses the prod build.

But to verify, I logged autoprefixer info at both project root and inside vite config and got the same output.
![](https://user-images.githubusercontent.com/9084735/184906465-a0b38648-d9e1-4dd9-b43f-d7dd0fbd747e.png)
![](https://user-images.githubusercontent.com/9084735/184906344-a2b995a6-6d32-4e7e-bc70-9b58ce85ae14.png)

Reply:
Huh, IE11, when it is removed in v1 branch.